bqetl_project.yaml

```
@@ -131,6 +131,7 @@ dry_run:
   - sql/moz-fx-data-shared-prod/contextual_services_derived/request_payload_tiles_v2/query.sql
   - sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_revenue_levers_daily_v1/query.sql
   - sql/moz-fx-data-marketing-prod/acoustic/**/*.sql
+  - sql/moz-fx-data-shared-prod/acoustic_external/contact_raw_v1/query.sql
```

```suggestion
  - sql/moz-fx-data-shared-prod/acoustic_external/**/query.sql
```

We should probablty include everything under `acoustic_external` as all of these will be accessing Fivetran project.